buildstockbatch/schemas/v0.3.yaml

```diff
@@ -44,6 +44,7 @@ gcp-ar-spec:
 gcp-job-environment-spec:
   vcpus: int(min=1, max=224, required=False)
   memory_mib: int(min=512, required=False)
+  boot_disk_mib: int(required=False)
```

Comment:
Can you add the default as a comment? also is this extra boot disk?

Reply:
> Can you add the default as a comment?

We're not generally documenting parameters here (but in the `docs/project_defn.rst` documentation file).

> also is this extra boot disk?

This _may_ add extra size the to the boot disk, more on this in [the other thread](https://github.com/rewiringamerica/buildstockbatch/pull/57#discussion_r1447947833).